### PR TITLE
Add btrees with different orders to test which order is most optimal with the current benchmarks

### PR DESCRIPTION
**Pull request by ByronBecker**
Thought this might be interesting to how BTrees of different orders perform with the benchmark.

Also, it might be useful once DTS is included and you go beyond inserting 50k objects. I think 4X DTS is now live in dfx >= 0.13.0

### CODE REVIEW

**Comment by ByronBecker** [comment]
Tagging @chenyan-dfinity for perf task signoff

**Comment by ByronBecker** [comment]
Also, these tests were out of curiosity in response to https://panthema.net/2007/stx-btree/stx-btree-0.8.3/doxygen-html/speedtest.html

(obviously those test results are running at much higher insertion levels that we'd expect in this scenario).

File: collections/motoko/src/btreemap_order_128.mo [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import Map "mo:btreemap/BTree";
+import Nat32 "mo:base/Nat32";
+import Iter "mo:base/Iter";
+import Option "mo:base/Option";
+import Random "random";
+
+actor {
```
Comment (chenyan-dfinity) [inline collections/motoko/src/btreemap_order_128.mo:7]: Can we make this as an `actor class` that takes `order` as init args? Then we can tune the number in `perf.sh`.

**Review by chenyan-dfinity — COMMENTED** [review]
Thanks for the PR! The CI error is a bit strange. Will take a deeper look tomorrow.

File: collections/motoko/src/btreemap_order_128.mo (hunk 1 above)
Comment (ByronBecker) [reply collections/motoko/src/btreemap_order_128.mo]: I decided to just update the BTree to default to an order of 32 after some internal testing.

**Comment by chenyan-dfinity** [comment]
It's unfortunate that CI cannot report results on forked branch. Do you happen to know a fix?

**Comment by ByronBecker** [comment]
> It's unfortunate that CI cannot report results on forked branch. Do you happen to know a fix?

Sorry, I don't have that much experience with CI yet 😞

One thing you maybe could do is add an action that generates and commits the file, something like this https://stackoverflow.com/a/70645011/6016681

It would be nice to have the result file in the `main` branch anyways.

**Comment by chenyan-dfinity** [comment]
This is a nice idea. Committing to main also gives us the diff for free. I will give it a try. Thanks.

**Comment by chenyan-dfinity** [comment]
Hmm, committing the file is sub-optimal. Users have to pull after every push. Also the flamegraphs won't work unless we commit everything.

Can you try to change this line: https://github.com/dfinity/canister-profiling/blob/main/.github/workflows/perf.yml#L66, and remove the "&& github.event.pull_request.head.repo.full_name == github.repository" part?

**Review by chenyan-dfinity — APPROVED** [review]
I'm glad that CI comments work for forked PR now :)

**Comment by ByronBecker** [comment]
> I'm glad that CI comments work for forked PR now :)

Awesome, thanks for working through this! From my personal findings, order 32 seems to work pretty well universally given the heap size limitations of canister on the IC.

**Pull request merged**